docs/sphinx/pom.xml

```diff
@@ -0,0 +1,169 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0
+    http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.openmicroscopy</groupId>
+    <artifactId>ome-model</artifactId>
+    <version>5.3.2-SNAPSHOT</version>
+    <relativePath>../..</relativePath>
+  </parent>
+
+  <groupId>org.openmicroscopy</groupId>
+  <version>5.3.2-SNAPSHOT</version>
+
+  <artifactId>ome-model-docs</artifactId>
```

Comment:
As I was going through the release process of ome-model 5.4.0 and reviewing the artifacts, one question on the naming scheme following https://github.com/openmicroscopy/bioformats/pull/2701#issuecomment-268545866. Before making the final commitment, should we modify the artifact id to use the same grammatical form across the board @rleigh-codelibre ?